www/css/web.css

```diff
@@ -3,9 +3,19 @@
 body{
 	font-family: "Georgia", serif;
 	font-size: 18px;
+	margin: 0;
+	min-height: 100vh;
+	display: flex;
+	flex-direction: column;
```

Comment:
Flexbox is necessary because the content of a chapter, like for the title page, may be quite short in height and we need the header to stick to the bottom (https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/).